financial-connections/src/main/java/com/stripe/android/financialconnections/features/notice/PresentSheet.kt

```diff
@@ -14,11 +15,23 @@ internal interface PresentSheet {
 internal class RealPresentSheet @Inject constructor(
     private val navigationManager: NavigationManager,
     private val noticeSheetContentRepository: NoticeSheetContentRepository,
+    private val accountUpdateRequiredContentRepository: AccountUpdateRequiredContentRepository,
 ) : PresentSheet {
 
     override fun invoke(content: NoticeSheetContent, referrer: Pane) {
-        noticeSheetContentRepository.set(content)
-        val route = Destination.Notice(referrer = referrer)
-        navigationManager.tryNavigateTo(route)
+        when (content) {
+            is NoticeSheetContent.UpdateRequired -> {
+                accountUpdateRequiredContentRepository.set(content)
+                val route = Destination.AccountUpdateRequired(referrer = referrer)
+                navigationManager.tryNavigateTo(route)
+            }
+            else -> {
+                noticeSheetContentRepository.set(content)
+                val route = Destination.Notice(referrer = referrer)
+                navigationManager.tryNavigateTo(route)
+            }
+
+        }
```

Comment:
Merged the `UpdateRequired` interactor into `PresentNoticeSheet`, so that refactoring it in the future has less impact on the presentation layer.